dashboard/app/bug.html

```diff
@@ -37,6 +37,35 @@
 	<textarea id="log_textarea" readonly rows="25" wrap=off>{{printf "%s" .SampleReport}}</textarea><br>
 	{{end}}
 
+	{{if .FixBisections}}
+	<table class="list_table">
+		<caption>All fix bisections ({{len .FixBisections}}):</caption>
+		<thead>
+		<tr>
+			<th><a onclick="return sortTable(this, 'Manager', textSort)" href="#">Manager</a></th>
```

Comment:
Now thinking about this, perhaps we could add these crashes right to the main Crashes table. That would not clutter the page more and won't require people to look at 2 tables which contain almost the same.
Do you see any potential problems with this approach? Anything potentially confusing for users?

Reply:
Hmm so currently the reason fix bisection results are being shown is to avoid confusion related to the bug's LastTime.
So, when doing fix bisection if a crash occurs on ToT, we want to show that crash on the bug page. But the crash might not necessarily have the crash title as the crashes in the "Crashes" table. That might cause some confusion. 
Also there is some ease to be able to see the failed bisection attempts separately/clearly while rolling out fix bisection.
Maybe it is better to have 2 tables but then move the common part into templates.html.

Reply:
Right, now memory returns to me :)
Let's start with 2 tables then, but with common table format.

Reply:
Thanks! I've made this change and updated the PR.